DEPENDENCIES.md

```diff
@@ -0,0 +1,41 @@
+
+*This document lists all the various products/tools used in Pipeline Service along with their current versions. We also use this document to track the evolution of the products/tools as and when they are upgraded.*
+
+
+### **Products**
+
+| **Component**                 | **Version**                                                                                                                           | **Purpose**                                                                  | **Comments**                                                                                                                                                              |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OpenShift                     | 4.10                                                                                                                                  | Platform                                                                     | Upgrades to next versions 4.11 need to be tested and certified                                                                                                            |
```

Comment:
I find "certified" a bit strong of a commitment

Reply:
Changed it to 'tested and approved'.